src/theme/_layout.scss

```diff
@@ -63,6 +64,37 @@ ion-modal.template-popup-modal {
     z-index: -1 !important;
   }
 }
+
+// Nav stack modal
+ion-modal.stack-modal {
+  // Progressively stagger successive stack modals
+  --height: calc(var(--safe-area-height) - ((var(--stack-index) + 1) * 56px));
+  --width: 100%;
+  --border-radius: 24px;
+  --box-shadow: 0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1);
+  --backdrop-opacity: 0.5;
+
+  // Override defalut ion-modal behaviour of hiding backdrop for subsequent modals after first
+  &:not([data-stack-index="0"]) {
+    --backdrop-opacity: 0.2 !important;
+  }
+
+  &::part(backdrop) {
+    background-color: black;
+  }
+
+  &::part(content) {
+    border-radius: var(--border-radius) var(--border-radius) 0 0;
+    position: absolute;
+    bottom: 0px;
+  }
+
+  ion-toolbar {
+    --background: var(--ion-color-background);
+    --color: var(--ion-color-primary);
+  }
+}
+
```

Comment:
nit(blocking): this feels like a lot of component-specific CSS in what should be a more general layout override. I know the styles need to be hoisted out of the component to apply at the injected modal level, although I think it would be worth trying to introduce a different pattern to do so.

How about we define global style overrides within a separate scoped scss file, e.g. `src\app\feature\stack\components\stack\stack.component.global.scss`, and then use the existing global deployment scss files (or a similar file within) to reference. E.g. 
```scss
@use(..\..\..\app\feature\stack\components\stack\stack.component.global.scss
```

Reply:
Good idea, I've refactored along these lines in https://github.com/IDEMSInternational/open-app-builder/pull/2550/commits/32650368240a12e732ecdb76c9b1185ef959fc74